tests/rng_test.cpp

```diff
@@ -0,0 +1,63 @@
+#include "catch/catch.hpp"
+
+#include "test_statistics.h"
+
+#include "rng.h"
+
+static void check_remainder( float proportion )
+{
+    statistics<float> stats;
+    const epsilon_threshold confidence_level{ proportion, 0.05 };
```

Comment:
I don't think `confidence_level` is a good name for these variables.  The confidence level is the thing set in the `stats` constructor.  Better something like `target_range`.